docs/application-connector/docs/044-remote-evironment-custom-resource.md

```diff
@@ -0,0 +1,98 @@
+---
+title: RemoteEnvironment
+type: Custom Resource
+---
+
+The `remoteenvironments.applicationconnector.kyma-project.io` Custom Resource Definition (CRD) is a detailed description of the kind of data and the format used to register a Remote Environment (RE) in Kyma. The `RemoteEnvironment` Custom Resource defines the APIs that the RE offers. After creating a new Custom Resource for a given RE, the RE is mapped to appropriate ServiceClasses in the Service Catalog. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd remoteenvironments.applicationconnector.kyma-project.io -o yaml
+```
+
+## Sample custom resource
+
+This is a sample resource that registers the `re-prod` Remote Environment which offers one service.
+
+```
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: RemoteEnvironment
+metadata:
+  clusterName: ""
+  creationTimestamp: 2018-09-25T12:24:41Z
+  generation: 1
+  labels:
+    app: ec-default-gateway
+    heritage: Tiller-gateway
+    release: ec-default-gateway
+  name: ec-default
+  namespace: ""
+  resourceVersion: "58761"
+  selfLink: /apis/applicationconnector.kyma-project.io/v1alpha1/remoteenvironments/ec-default
+  uid: fa2d9272-c0bd-11e8-ad1b-000d3a2fa0d4
+spec:
+  accessLabel: echo-access-label
+  description: This Remote Environment corresponds to the connected remote system.
+  labels: null
+  services:
+  - description: Events v1
+    displayName: Events v1
+    entries:
+    - credentialsSecretName: ""
+      gatewayUrl: ""
+      oauthUrl: ""
+      targetUrl: ""
+      type: Events
+    id: 40d3f17a-b376-4b02-8755-f24ceb76b27a
+    identifier: ec-all-events
+    labels:
+      connected-app: ec-default
+    longDescription: Events
+    name: ec-events-v1-bbd7f
+    providerDisplayName: Provider
+  - description: Commerce Webservices
+    displayName: Commerce Webservices
+    entries:
+    - accessLabel: re-ec-default-58cc45cd-b9ca-4c53-8019-0296774b7aa1
+      credentialsSecretName: re-ec-default-58cc45cd-b9ca-4c53-8019-0296774b7aa1
+      gatewayUrl: http://re-ec-default-58cc45cd-b9ca-4c53-8019-0296774b7aa1.kyma-integration.svc.cluster.local
+      oauthUrl: https://oauth/token
+      targetUrl: https://rest/v2
+      type: API
+    id: 58cc45cd-b9ca-4c53-8019-0296774b7aa1
+    identifier: commercewebservices
+    labels:
+      connected-app: ec-default
+    longDescription: RealLife E-Commerce Webservices v2
+    name: rlife-e-commerce-webservices-v2-a4508
+    providerDisplayName: Provider
+```
+
+## Custom resource parameters
```

Comment:
there must be a clear information which spec fields are managed by the Metadata API and user should not touch them on CR level because it will anyway not work.
Maybe separate table, or additional column or additional sentence in description of each field

Reply:
added information about the fields that should not be edited